src/components/mainMenu.vue

```diff
@@ -34,13 +34,19 @@ export default defineComponent({
       lang,
       launchOnStart: false,
       autoLauncher: global.autoLauncher,
-      disableAutoLaunch: false
+      disableAutoLaunch: false,
+      devMode: false
     }
   },
   mounted() {
+    this.checkDev()
     this.initMenu()
   },
   methods: {
+    checkDev() {
+      if (util.CONTEXT_MENU != "OFF")
```

Comment:
do we use `devMode` anywhere else? I guess we can just use `util.CONTEXT_MENU !== "OFF"` in `v-if=` and skip addiotional method and state

Reply:
sure

Reply:
no can do, since util is defined below vue part